modules/src/openai_proxy/item.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+categories:
+- genai
+description: OpenAI application runtime based on fastapi
+example: openai.ipynb
+generationDate: 2025-11-11:12-25
+hidden: false
+labels:
+  author: Iguazio
+mlrunVersion: 1.10.0
+name: openai_proxy_app
```

Comment:
great, but also need to update the py and ipynb files names accordingly:
- openai_proxy_app.py
- openai_proxy_app.ipynb